### PR TITLE
Update proton-ge-custom-src.tar.xz to GE-Proton7-8-2

### PR DESCRIPTION
**Pull request by ColinKinloch**
Hopefully fixed tarball.
Changes for proton 7 from the Proton-Exp flathub repo.

### CODE REVIEW

**Comment by Lctrs** [comment]
Could you also remove all other releases in `com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml` ?

File: com.valvesoftware.Steam.CompatibilityTool.Proton.yml [hunk 1]
```diff
@@ -36,6 +40,10 @@ x-compat-i386-opts: &compat_i386_opts
     CXX: ccache i686-unknown-linux-gnu-g++
     PKG_CONFIG_PATH: /app/share/steam/compatibilitytools.d/Proton-GE/lib32/pkgconfig:/app/lib32/pkgconfig:/app/share/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig:/usr/share/pkgconfig
     LD_RUN_PATH: /app/share/steam/compatibilitytools.d/Proton-GE/lib32
+    FLATPAK_LIBDIR: /app/share/steam/compatibilitytools.d/Proton/lib32
+    FLATPAK_LIBDIR_WINE: /app/share/steam/compatibilitytools.d/Proton/lib32/wine
+    FLATPAK_LIBDIR_WINE_ELF: /app/share/steam/compatibilitytools.d/Proton/lib32/wine/i386-unix
+    FLATPAK_LIBDIR_WINE_PE: &lib32_wine_pe /app/share/steam/compatibilitytools.d/Proton/lib32/wine/i386-windows
```
Comment (Lctrs) [inline com.valvesoftware.Steam.CompatibilityTool.Proton.yml:46]:
```suggestion
    FLATPAK_LIBDIR: /app/share/steam/compatibilitytools.d/Proton-GE/lib32
    FLATPAK_LIBDIR_WINE: /app/share/steam/compatibilitytools.d/Proton-GE/lib32/wine
    FLATPAK_LIBDIR_WINE_ELF: /app/share/steam/compatibilitytools.d/Proton-GE/lib32/wine/i386-unix
    FLATPAK_LIBDIR_WINE_PE: &lib32_wine_pe /app/share/steam/compatibilitytools.d/Proton-GE/lib32/wine/i386-windows
```

File: com.valvesoftware.Steam.CompatibilityTool.Proton.yml [hunk 2]
```diff
@@ -21,6 +21,10 @@ build-options:
     PKG_CONFIG_PATH: /app/share/steam/compatibilitytools.d/Proton-GE/lib/pkgconfig:/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig:/usr/share/pkgconfig
     LD_RUN_PATH: /app/share/steam/compatibilitytools.d/Proton-GE/lib
     PYTHONPATH: /app/share/steam/compatibilitytools.d/Proton-GE/lib/python3.9/site-packages
+    FLATPAK_LIBDIR: /app/share/steam/compatibilitytools.d/Proton/lib
+    FLATPAK_LIBDIR_WINE: /app/share/steam/compatibilitytools.d/Proton/lib/wine
+    FLATPAK_LIBDIR_WINE_ELF: /app/share/steam/compatibilitytools.d/Proton/lib/wine/x86_64-unix
+    FLATPAK_LIBDIR_WINE_PE: /app/share/steam/compatibilitytools.d/Proton/lib/wine/x86_64-windows
```
Comment (Lctrs) [inline com.valvesoftware.Steam.CompatibilityTool.Proton.yml:27]:
```suggestion
    FLATPAK_LIBDIR: /app/share/steam/compatibilitytools.d/Proton-GE/lib
    FLATPAK_LIBDIR_WINE: /app/share/steam/compatibilitytools.d/Proton-GE/lib/wine
    FLATPAK_LIBDIR_WINE_ELF: /app/share/steam/compatibilitytools.d/Proton-GE/lib/wine/x86_64-unix
    FLATPAK_LIBDIR_WINE_PE: /app/share/steam/compatibilitytools.d/Proton-GE/lib/wine/x86_64-windows
```

**Review by Lctrs — CHANGES_REQUESTED** [review]

**Comment by ColinKinloch** [comment]
Working with Elden Ring online :+1:

**Comment by ColinKinloch** [comment]
bot, build com.valvesoftware.Steam.CompatibilityTool.Proton-GE

**Comment by Lctrs** [comment]
Thank you @ColinKinloch ! Good work !

**Pull request merged**